Applications/Piano/PianoWidget.h

```diff
@@ -55,4 +59,15 @@ class PianoWidget final : public GWidget {
     bool m_release_enabled { false };
 
     bool keys[256];
+
+#define key_count sizeof(piano_keys) / sizeof(PianoKey)
```

Comment:
Let's use a `static const size_t` or similar for this instead of putting a macro in the global preprocessor namespace.

Reply:
No longer relevant.